### ISSUE TITLE
assert correctness of output

### ISSUE THREAD

**Issue by neonstalwart** [vdom-benchmark/vdom-benchmark-base#2]
it's one thing to be fast but it's irrelevant if you're wrong.  

in virtual-dom we've recently been addressing some issues in our reordering logic that more than likely would have produced incorrect output for some of your benchmarks.  unfortunately, we didn't have sufficient tests to pick up on these issues but if there had been assertions in these benchmark tests we may have been aware of these issues earlier - this is a benefit to the producers of the libraries.  for consumers, the benefit is that in addition to seeing how the libraries perform, they can have some indication of the correctness of the output of these libraries too.

i realize it would add to the time it takes to run tests and maybe it could be a configurable option rather than the default but i think it would be of great value.

**Comment by localvoid** [comment]
Yes, it would be great to check for correctness, but there are only simple use cases in this benchmark that most likely covered by unit tests in all libraries, so is it really worth it?
